src/lib/components/forms/radio-buttons.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+  export let options: string[];
+  let selectedOption: string;
+</script>
+
+<div>
```

Comment:
```suggestion
<fieldset>
```
I think this fits well